app/mailers/admin_mailer.rb

```diff
@@ -41,11 +41,20 @@ def lightspeed_notification_email(organization, api_key)
     mail(to: "admin@bikeindex.org", subject: "Api Notification sent!")
   end
 
-  def theft_alert_purchased(theft_alert)
+  def theft_alert_purchased(theft_alert, notify_of_recovery: false)
     @theft_alert = theft_alert
     @theft_alert_plan = theft_alert.theft_alert_plan
     @creator = theft_alert.creator
     @bike = theft_alert.bike
-    mail(to: "stolenbikealerts@bikeindex.org", subject: "Theft Alert purchased: #{@theft_alert.id}")
+
+    if notify_of_recovery
```

Comment:
This confused me at first, that both emails where in the same mailer method

Reply:
I would like it to be made clearer, somehow?

I think being in the same method makes sense. Idea for how to resolve:  mailer method was renamed `theft_alert_notification`, second arg was `notification_type: "purchased"`

My

Reply:
Good call thanks — updated in 1998c27d